src/vm_cpu.rs

```diff
@@ -149,6 +154,81 @@ impl VmPmuState {
             None
         }
     }
+
+    fn gen_counter_mask(high: u64, low: u64) -> u64 {
+        (u64::MAX << low) & (u64::MAX >> (63 - high))
+    }
```

Comment:
This mask generation function can also be used in other places in salus. Move it a common place ?

Reply:
Possibly, but can't think of any convenient place right now?

Reply:
There are common no bitops utils source file in salus ? In Linux kernel, there is a GENMASK macro that does the same thing and widely used.

Reply:
We haven't had such a need for it yet. I'm sure there's a 3rd-party crate for it, but not sure we'd want to pull it in just for this.